dev/CommandBarFlyout/CommandBarFlyoutCommandBar.cpp

```diff
@@ -965,7 +981,7 @@ bool CommandBarFlyoutCommandBar::IsControlFocusable(
     return control &&
         control.Visibility() == winrt::Visibility::Visible &&
         (control.IsEnabled() || control.AllowFocusWhenDisabled()) &&
-        (!checkTabStop || control.IsTabStop());
+        (control.IsTabStop() || (!checkTabStop && !control.try_as<winrt::AppBarSeparator>())); // AppBarSeparator is not focusable if IsTabStop is false
```

Comment:
>// AppBarSeparator is not focusable if IsTabStop is false [](http://example.com/codeflow?start=95&length=57)

As in my other comment - this is true of anything, not just separators, so hopefully you don't need to special case that type

Reply:
I repro'd the problem @licanhua and applied your fix - it does look reasonable to me. Thanks.